tests/10-bbs-create.js

```diff
@@ -23,138 +21,115 @@ const should = chai.should();
-            it('Dereferencing "verificationMethod" MUST result in an object ' +
-              'containing a type property with "Multikey" value.',
-            async function() {
-              this.test.cell = {
-                columnId: `${name}: ${keyType}`, rowId: this.test.title
-              };
-              verificationMethodDocuments.should.not.eql([], 'Expected ' +
-                'at least one "verificationMethodDocument".');
-              verificationMethodDocuments.some(
-                verificationMethodDocument =>
-                  verificationMethodDocument?.type === 'Multikey'
-              ).should.equal(true, 'Expected at least one proof to have ' +
-                '"type" property value "Multikey".'
-              );
-            });
-            it('The "publicKeyMultibase" property of the verification method ' +
-              'MUST be public key encoded according to MULTICODEC and ' +
-              'formatted according to MULTIBASE.', async function() {
-              this.test.cell = {
-                columnId: `${name}: ${keyType}`, rowId: this.test.title
-              };
-              verificationMethodDocuments.should.not.eql([], 'Expected ' +
-                '"verificationMethodDocuments" to not be empty.');
-              verificationMethodDocuments.some(
-                verificationMethodDocument => {
-                  const multibase = 'z';
-                  const {publicKeyMultibase} = verificationMethodDocument;
-                  return publicKeyMultibase.startsWith(multibase) &&
-                    shouldBeBs58(publicKeyMultibase) &&
-                    shouldBeMulticodecEncoded(publicKeyMultibase);
-                }
-              ).should.equal(true, 'Expected at "publicKeyMultibase" to ' +
-                'be MULTIBASE formatted and MULTICODEC encoded.');
-            });
+      describe(name, function() {
+        const [issuer] = endpoints;
+        const verifier = implementation.verifiers.filter(
+          v => v.tags.has(tag));
+        let issuedVc;
+        let proofs;
+        const verificationMethodDocuments = [];
+        before(async function() {
+          issuedVc = await createInitialVc({issuer, vc});
+          proofs = Array.isArray(issuedVc?.proof) ? issuedVc.proof :
+            [issuedVc?.proof];
+          // FIXME: Update documentLoader to support BLS12-381 cryptographic key
+          // const verificationMethods = proofs.map(
+          //   proof => proof.verificationMethod);
+          // for(const verificationMethod of verificationMethods) {
+          //   const verificationMethodDocument = await documentLoader({
+          //     url: verificationMethod
+          //   });
+          //   verificationMethodDocuments.push(verificationMethodDocument);
+          // }
+        });
+        it('The field "cryptosuite" MUST be "bbs-2023".', function() {
+          this.test.cell = {
+            columnId: name, rowId: this.test.title
+          };
```

Comment:
This can be addressed later (rather than in this PR, if you would prefer), but I wonder if we could set this generally for all tests here, and then override if it ever varies.

Reply:
Moved the setup within `beforeEach` block [1d484ea](https://github.com/w3c-ccg/vc-di-bbs-test-suite/pull/3/commits/1d484ea44845a7fbc36055bee67e3a90c5a205a9)